hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -721,4 +721,24 @@ private Constants() {
    * Default change detection require version: true.
    */
   public static final boolean CHANGE_DETECT_REQUIRE_VERSION_DEFAULT = true;
+
+  public static final String MULTIPART_DOWNLOAD_ENABLED =
```

Comment:
should add javadoc for each of these, and add to core-default.xml

Reply:
Some documentation should be added to hadoop-aws/index.md and hadoop-aws/performance.md about this capability.